ui/app/templates/components/form-field-from-model.hbs

```diff
@@ -0,0 +1,84 @@
+<div class="field">
```

Comment:
Doesn't really work this time because there's no corresponding js file, but one day I'd like to have JSDocs or the like describing what type of attributes this accepts and what it does. 
Also now that it's a component maybe we should add a test?

Reply:
Good call.  I'll work on the test.